src/core_modules/capture-core-utils/date/date.utils.js

```diff
@@ -1,17 +1,25 @@
 // @flow
-import moment from '../moment/momentResolver';
+import { moment } from '../moment';
```

Comment:
could we do that instead? 
```suggestion
import moment from 'moment';
```

Reply:
When we started using moment in our app, there was an issue with moment causing the global locale setting not be preserved without this workaround (i.e. the global moment locale would always be 'en' even though we set it to something else). Had a quick look at our app code now and since the direct node module import is used in some places previously causing problems, the issue is most likely resolved. But to keep our code  consistent (for the most part) and be able to verify that the issue is resolved, let's keep the import like it is for now and do this update for the entire app through a maintenance task.